config/base_boxes.yaml

```diff
@@ -8,20 +8,16 @@ shells:
 
 boxes:
   centos6:
-    box_name:   'centos6'
+    box_name:   'centos/6'
```

Comment:
By removing the URL's, and specifying the box name like this, we'll get the box from Vagrant atlas instead: https://atlas.hashicorp.com/boxes/search

The boxes we have here are ancient, using centos 6.4 and 7.1 - we should use newer (and need newer, right now failing for me on the 6.4 box)